build.xml

```diff
@@ -20,6 +20,57 @@
     </target>
 -->
 
+  <scriptdef name="scepcificTest" language="javascript">

```

Comment:
Typo in name: "scepcificTest". Probably best to name this script as a verb (e.g., "runSpecifiedTests").